_api_app/app/Configuration/SiteSettingsConfigService.php

```diff
@@ -7,6 +7,7 @@
 class SiteSettingsConfigService
 {
     private $settings;
+    private $defaultTemplate = 'messy-0.4.2';
```

Comment:
why not simply put this in the default value field in settings file? `engine/inc.settings.php:57`

Reply:
Already tried, old Berta doesn't generate `settings.xml` file in case it's defined.

Reply:
Then please add a comment why this is done here.